### PR TITLE
KAFKA-8835: KIP-352 docs update

### PR DESCRIPTION
**Pull request by viktorsomogyi**
Doc updates for KIP-352

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by viktorsomogyi** [comment]
#7361 should be merged first

**Comment by viktorsomogyi** [comment]
Test runs were flaky (and since no java code was modified it's ok)

File: docs/ops.html [hunk 1]
```diff
@@ -866,7 +866,7 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td></td>
       </tr>
       <tr>
-        <td># of under replicated partitions (|ISR| &lt |all replicas|)</td>
+        <td># of under replicated partitions (|original replicas| - |ISR| &gt 0)</td>
```
Comment (stanislavkozlovski) [inline docs/ops.html]: What are people's thoughts if we say `non-reassigning replicas`?

From a first read of the docs, I don't believe people will understand what "original replicas" means

File: docs/ops.html [hunk 2]
```diff
@@ -1069,6 +1069,31 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
...
+        <td>kafka.server:type=ReplicaManager,name=ReassignmentMaxLagOnFollower</td>
+        <td>Zero if there is no reassignment but should converge to zero if there is an ongoing reassignment.</td>
+      </tr>
+      <tr>
+        <td>Outgoing rate of reassignment</td>
+        <td>kafka.server:type=BrokerTopicMetrics,name=ReassignmentBytesOutPerSec</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>Incoming rate of reassignment</td>
```
Comment (stanislavkozlovski) [inline docs/ops.html]: Would `incoming bytes rate of reassignment traffic` be more descriptive?

File: docs/ops.html [hunk 3]
```diff
@@ -1069,6 +1069,31 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td>kafka.server:type=transaction-coordinator-metrics,name=partition-load-time-avg</td>
         <td>average time, in milliseconds, it took to load transaction metadata from the consumer offset partitions loaded in the last 30 seconds</td>
       </tr>
+      <tr>
+        <td>Number of reassigning partitions</td>
+        <td>kafka.server:type=ReplicaManager,name=ReassigningPartitions</td>
+        <td>The number of reassigning leader partitions on a broker.</td>
+      </tr>
+      <tr>
+        <td>Maximum reassignment lag on leader side</td>
```
Comment (stanislavkozlovski) [inline docs/ops.html]: To be consistent with the docs on `kafka.server:type=ReplicaFetcherManager,name=MaxLag`, could we say `lag in messages`

File: docs/ops.html (hunk 1 above)
Comment (viktorsomogyi) [reply docs/ops.html]: Thanks, I was trying to find a better term, I think this is it.

File: docs/ops.html [hunk 4]
```diff
@@ -866,7 +866,7 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td></td>
       </tr>
       <tr>
-        <td># of under replicated partitions (|ISR| &lt |all replicas|)</td>
+        <td># of under replicated partitions (|non-reassigning replicas| - |ISR| &gt 0)</td>
```
Comment (stanislavkozlovski) [inline docs/ops.html]: Should we reword this to "the number of non-reassigning replicas - the number of ISR" to make it explicit that something like
```
r=[1,2,3,4,5,6]
target_replicas=[4,5,6]
isr=[3,4,5]
```
Would not count as URP?

Not sure if people may read the above exaple as `[1,2,3] - [3,4,5] > 0` given the current wording

Comment (viktorsomogyi) [reply docs/ops.html]: Cool, I'll reword that.

**Review by stanislavkozlovski — APPROVED** [review]

**Comment by viktorsomogyi** [comment]
@hachikuji would you please review this too?

**Comment by viktorsomogyi** [comment]
@hachikuji could you please give this a look and merge if possible?

**Comment by viktorsomogyi** [comment]
retest this please

**Comment by viktorsomogyi** [comment]
retest this please

**Comment by viktorsomogyi** [comment]
@hachikuji could you please merge this into trunk? I'll make another PR for the 2.5 branch

**Comment by viktorsomogyi** [comment]
retest this please

**Review by hachikuji — APPROVED** [review]
LGTM
